hicommonwealth/commonwealth
# Fix Snapshot mobile link

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/5727

## Description of Changes
- Added a loader for when snapshot proposal request is loading, will look like this

https://github.com/hicommonwealth/commonwealth/assets/51641047/73adad3b-ec84-47a6-b179-8d7c379290c9




- Fixed CQ in the updated file

## "How We Fixed It"

By adding a loader state

## Test Plan

To confirm if the issue is fixed on the thread (for which the issue was reported)
- Visit https://commonwealth.im/linksdao/discussion/10108-proposal-17-linksdao-golf-course-purchase-scotland
- Verify that the "linked proposals" section shows a cow bobber when request to https://hub.snapshot.org/graphql is pending in network tab
- Verify the bobber goes away after request is finished successfully and you see the snapshot link, that correctly navigates

## Deployment Plan
N/A

## Other Considerations
N/A
